Example/Backpack/Utils/FeatureStories/Cells/CellsDataSource.swift

```diff
@@ -26,6 +26,12 @@ struct Components {
         let title: String
         let onSelection: () -> Void
     }
+    
+    struct Tab {
+        let title: String
+        let viewController: UIViewController
```

Comment:
This makes that every view controller is created at startup, should this be a method rather? like just `let viewController: () -> UIViewController`?